input/fsh/Terminology.fsh

```diff
@@ -390,14 +390,14 @@ Description: "Valueset of the Functional Assessment of Cancer Therapy questionna
 
 
 // SURVIVAL AND DISEASE CONTROL 
-// ValueSet: RecurrenceMethod
-// Id: RecurrenceMethod
-// Title: "Method of confirming recurrence"
-// Description: "Valueset of the methods used to confirm recurrence of breast cancer"
-// * SCT#394914008 "Radiological diagnosis"
-// * SCT#67151002 "Histological diagnosis"
-// both --> option to use postcoordinated terms
-// * NullFlavor#UNK "unknown"
+ValueSet: RecurrenceMethod
+Id: RecurrenceMethod
+Title: "Method of confirming recurrence"
+Description: "Valueset of the methods used to confirm recurrence of breast cancer"
+* SCT#394914008 "Radiological diagnosis"
+* SCT#67151002 "Histological diagnosis"
+//both --> option to use postcoordinated terms
```

Comment:
ICHOM hasn't provided a postcoordinated term here unfortunately - so let's define a codesystem just for this variable and use the number code. Here's an example - https://github.com/HL7/fhir-ichom-breast-cancer-ig/pull/22/files#diff-7b411e464cdc7b4cc2847257f3d596aa64d494b149a99ce2b628997c080c846aR428